modules/etcd-increase-db.adoc

```diff
@@ -0,0 +1,180 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="etcd-increase-db_{context}"]
+= Increasing the database size for etcd
+
+You can set the disk quota in gigabytes for each etcd instance. If you set a disk quota for your etcd instance, the instance can increase up to the value that you specify.
```

Comment:
```suggestion
You can set the disk quota in gigabytes for each etcd instance. 
```

Reply:
This sentence can be removed. The next sentences clarify that the value can be only increasing and set upto int 32.